dod.py

```diff
@@ -1,14 +1,34 @@
 import pandas as pd
 import numpy as np
+import requests
 import datetime
 import pprint
+from us_state_abbrev import us_state_abbrev
 
 # @TODO: Attempt today. If that fails, attempt yesterday.
 latest = datetime.date.today() - datetime.timedelta(days=1)
 
+interventions_url = 'https://raw.githubusercontent.com/covid-projections/covid-projections/master/src/assets/data/interventions.json'
+interventions = requests.get(interventions_url).json()
+interventions_df = pd.DataFrame(
+    list(interventions.items()),
+    columns=['state', 'intervention']
+)
+interventions_df.info()
+
+abbrev_df = pd.DataFrame(
+    list(us_state_abbrev.items()),
+    columns=['state', 'abbreviation']
+)
+abbrev_df.info()
+
+projections_df = pd.read_csv('projections_03-26-2020.csv')
+projections_df.info()
+
 url = 'https://github.com/CSSEGISandData/COVID-19/raw/master/csse_covid_19_data/csse_covid_19_daily_reports/{}.csv'.format(
```

Comment:
up to this point, it looks like it's mostly obtaining data.  i can definitely see a helper method.